.github/workflows/publish-sdist-wheels.yml

```diff
@@ -45,7 +45,11 @@ jobs:
       - name: Build wheels Mac OS
         if: runner.os == 'macOS'
         env:
-          CIBW_BEFORE_BUILD: 'brew update; brew unlink gcc@8 gcc@9; brew install cmake hdf5'
+          CIBW_BEFORE_BUILD: |
```

Comment:
is there a behavior change here?

Reply:
No, I think the only change was in brew itself.

Reply:
k, thanks.